### PR TITLE
isEqualTo and objectParent Optimisation

### PR DESCRIPTION
**Pull request by Jawshy**
Resolves https://github.com/AsYetUntitled/Framework/issues/166. 

#### Changes proposed in this pull request: 
- Essentially replaces `x != y` with `!(x isEqualTo y)`. 
- Replaced `!(vehicle player isEqualTo player)` with `!(isNull objectParent player)`. 

**Not tested in the slightest.** Any help checking and testing the changes would be greatly appreciated. Do note that due to `isEqualTo` performing case sensitive comparison on [strings](https://community.bistudio.com/wiki/String) it cannot substitute `!=` in all cases.

### CODE REVIEW

File: Altis_Life.Altis/core/functions/fn_fetchCfgDetails.sqf [hunk 1]
```diff
@@ -98,7 +98,7 @@ switch (_section) do
         if (!isNil "_muzzles") then {
             private ["_tmp"];
             {
```
Comment (BoGuu) [inline Altis_Life.Altis/core/functions/fn_fetchCfgDetails.sqf]: Marked

File: Altis_Life.Altis/core/pmenu/fn_cellphone.sqf [hunk 2]
```diff
@@ -21,15 +21,15 @@ if (FETCH_CONST(life_adminlevel) < 1) then {
...
 };
 {
-    if (alive _x && _x != player) then {
-        _type = switch (side _x) do {
-            case west: {"Cop"};
-            case civilian: {"Civ"};
-            case independent: {"Med"};
+    if (alive _x && !(_x isEqualTo player)) then {
+        switch (side _x) do {
+            case west: {_type = "Cop"};
```
Comment (BoGuu) [inline Altis_Life.Altis/core/pmenu/fn_cellphone.sqf]: Ahh

File: life_hc/MySQL/WantedSystem/fn_wantedAdd.sqf [hunk 3]
```diff
@@ -90,8 +90,8 @@ if (count _queryResult != 0) then
     _query = format ["UPDATE wanted SET wantedCrimes = '%1', wantedBounty = wantedBounty + '%2', active = '1' WHERE wantedID='%3'",_pastCrimes,_val,_uid];
     [_query,1] call HC_fnc_asyncCall;
 } else {
-    _crime = [_type select 0];
-    _crime = [_crime] call HC_fnc_mresArray;
-    _query = format ["INSERT INTO wanted (wantedID, wantedName, wantedCrimes, wantedBounty, active) VALUES('%1', '%2', '%3', '%4', '1')",_uid,_name,_crime,_val];
+    _crimes = [_type select 0];
+    _crimes = [_crimes] call HC_fnc_mresArray;
```
Comment (BoGuu) [inline life_hc/MySQL/WantedSystem/fn_wantedAdd.sqf]: Conflicts

File: life_server/Functions/MySQL/fn_mresArray.sqf [hunk 4]
```diff
@@ -15,7 +15,7 @@ _array = toArray(_array);
 for "_i" from 0 to (count _array)-1 do
 {
     _sel = _array select _i;
-    if ((_i != 0 && _i != ((count _array)-1))) then
+    if ((!(_i isEqualTo 0) && _i != ((count _array)-1))) then
```
Comment (BoGuu) [inline life_server/Functions/MySQL/fn_mresArray.sqf]: !=

File: life_server/Functions/WantedSystem/fn_wantedAdd.sqf [hunk 5]
```diff
@@ -36,6 +36,10 @@ private _val = [_type select 1] call DB_fnc_numberSafe;
 private _number = _type select 0;
 
 if !(count _queryResult isEqualTo 0) then {
+    _crime = format ["SELECT wantedCrimes, wantedBounty FROM wanted WHERE wantedID='%1'",_uid];
+    _crimeresult = [_crime,2] call DB_fnc_asyncCall;
+    _pastcrimess = [_crimeresult select 0] call DB_fnc_mresToArray;
```
Comment (BoGuu) [inline life_server/Functions/WantedSystem/fn_wantedAdd.sqf]: Conflicts

File: Altis_Life.Altis/core/functions/fn_fetchCfgDetails.sqf (hunk 1 above)
Comment (BoGuu) [inline Altis_Life.Altis/core/functions/fn_fetchCfgDetails.sqf]: Could be an issue with case

**Comment by Jawshy** [comment]
Changes applied. This should now be ready to merge, @BoGuu.

**Comment by BoGuu** [comment]
LGTM

**Pull request merged**